.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish Pipeline-ai 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
+
+jobs:
+  build-and-publish:
+    name: Build and publish Pipeline-ai 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.9
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: >-
+        python -m
+        build
+        --sdist
+        --wheel
+        --outdir dist/
+        .
+    - name: Publish distribution 📦 to Test PyPI
```

Comment:
What version is used for these development builds? If we have two concurrent PRs will their workflows end up overwriting the same version on Test PyPI?

Reply:
I added a boolean value allowing duplicate versions to be deployed to test-pypi as the idea is to have constant deployments there, but the actual pypi will throw if this case happens

Reply:
removed to avoid duped versions